src/workerd/server/workerd.c++

```diff
@@ -703,6 +703,7 @@ class CliMain final: public SchemaFileImpl::ErrorReporter {
           .addSubCommand("test", KJ_BIND_METHOD(*this, getTest), "run unit tests")
           .addSubCommand("pyodide-lock", KJ_BIND_METHOD(*this, getPyodideLock),
               "outputs the package lock file used by Pyodide")
+          .addSubCommand("make-pyodide-snapshot", KJ_BIND_METHOD(*this, getMakePyodideSnapshot), "Make a Pyodide memory snapshot")
```

Comment:
If it's possible to suppress this from the `--help` text that might be a good idea since I don't expect it to be particularly useful for downstream users.